galaxy_importer/loaders/collection.py

```diff
@@ -99,6 +105,58 @@ def load(self):
             requires_ansible=self.requires_ansible,
         )
 
+    def _lint_collection(self):
+        """Log ansible-lint output.
+
+        ansible-lint stdout are linter violations, they are logged as warnings and errors,
+        depending on the rule level.
+
+        ansible-lint stderr includes info about vars, file discovery,
+        summary of linter violations, config suggestions, and raised errors.
+        Only raised errors are logged, they are logged as errors.
+        """
+
+        self.log.info("Linting collection via ansible-lint...")
+
+        if not shutil.which("ansible-lint"):
+            self.log.warning("ansible-lint not found, skipping lint of collection")
+            return
+
+        cmd = [
+            "/usr/bin/env",
+            f"ANSIBLE_LOCAL_TEMP={self.cfg.ansible_local_tmp}",
+            "ansible-lint",
+            "--profile",
+            "production",
+            "--exclude",
+            "tests/integration/",
+            "--exclude",
+            "tests/unit/",
+            "--parseable",
+            "--nocolor",
+        ]
+        self.log.debug("CMD: " + " ".join(cmd))
+        proc = Popen(
+            cmd,
+            cwd=self.path,
+            encoding="utf-8",
+            stdout=PIPE,
+            stderr=PIPE,
+        )
+        proc.wait()
+
+        for line in proc.stdout:
+            if line.endswith("(warning)\n"):
```

Comment:
what are the stdout possibilites that do not end in `(warning)`?

Reply:
`ansible-lint` has an internal tagging system for rules that determines which ones are a 'warning' level and which ones are 'error' level. The 'warning' level is usually reserved for rules that are 'experimental', aka new to the rule list. 
Here's an example of a 'warning' level lint output that would be a line in stdout:
```tests/sanity/ignore-2.9.txt:139: sanity[cannot-ignore]: Ignore file contains pylint:consider-using-f-string at line 139, which is not a permitted ignore. (warning)```

Here's an example of an `error` level lint violation in stdout:
```playbooks/test-playbook.yml:36: name[play]: All plays should be named.```

The `(warning)` string at the end of the line is the best way I found to determine what violations aren't full errors as that is consistent for all warnings.

Reply:
@awcrosby I agree! Though when I'm thinking about it now, I'm not sure we're ready to fail import fully on ansible-lint or ansible-test sanity errors yet, and having them labeled as "warning" might communicate to partners that "warning" related-issues do not need to be resolved, vs if they were labeled with "error". I'll default to your judgement on that - but I could see us getting feedback from partners about "Why do we need to fix this if its just a warning?"

Reply:
We should probably route ansible-lint stdout into the log type of warning rather than error (that matches current functionality). As for those stdout that end in `(warning)`... those could either not be logged, or logged in the same way as lines without it.

Reply:
@awcrosby Now that you say that - just today, after we started talking about this, ansible-lint had a change that makes it highly unlikely that there will be any `(warning)` messages when calling the `--production profile`, so I think it's safe to remove that condition and set everything ansible-lint related to warning level.